tests/AssertTest.php

```diff
@@ -323,8 +328,6 @@ public function testAssert($method, $args, $success, $multibyte = false, $minVer
         }
         if ($multibyte && !function_exists('mb_strlen')) {
             $this->markTestSkipped('The function mb_strlen() is not available');
-
-            return;
```

Comment:
Thanks. 
But these has nothing todo with this PR. They should have been in a separate PR. 
(I do not request you to remove them, just a FYI for the future)